runtime/worker.rs

```diff
@@ -83,6 +84,32 @@ pub struct WorkerOptions {
   pub stdio: Stdio,
 }
 
+impl Default for WorkerOptions {
+    fn default() -> Self {
+        Self {
+            web_worker_preload_module_cb: Arc::new(|_| unimplemented!("web workers are not supported")),
+            create_web_worker_cb: Arc::new(|_| unimplemented!("web workers are not supported")),
```

Comment:
Crashing from a library is certainly always problematic, but this basically seems to be the recommended implementation for `deno_runtime` consumers based on what the CLI does.